.env.example

```diff
@@ -6,3 +6,9 @@ OPERATOR_KEY_ETH_SENDRAWTRANSACTION=
 CHAIN_ID=
 MIRROR_NODE_URL=
 E2E_RELAY_HOST=
+
+NETWORK_NAME=
```

Comment:
We should document these configs.
For now i'll settle for in line code comment when reading the new property. I"ll open a ticket to add a configuration table that explains items.

Also i really wanna move to yaml so we can group things better